site/js/modules/autograder/test/data/hw0_solution.py

```diff
@@ -0,0 +1,5 @@
+def function1():
+    return True
+
+def function2(val):
+    return val + 1
```

Comment:
Missing newline.

Note that this is not an extra newline, just the newline on line 5.